sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/internal/SubscribeTransform.java

```diff
@@ -56,7 +56,7 @@ public class SubscribeTransform extends PTransform<PBegin, PCollection<Sequenced
 
   private static final long MEBIBYTE = 1L << 20;
   private static final long SOFT_MEMORY_LIMIT = 512 * MEBIBYTE;
-  private static final long MIN_PER_PARTITION_MEMORY = 4 * MEBIBYTE;
+  private static final long MIN_PER_PARTITION_MEMORY = 10 * MEBIBYTE;
```

Comment:
LGTM but would this be better as a pipeline option or something that could be configured on PubsubLiteIO?  There may perhaps be cases where the user wants to specify higher per-partition min/max or total memory limit.

Not yet a committer though so someone else will have to merge

Reply:
No, it should really match the service max message size. I don't see this changing again in the future though